config/config.go

```diff
@@ -754,6 +754,7 @@ type RemoteWriteConfig struct {
 	WriteRelabelConfigs []*relabel.Config `yaml:"write_relabel_configs,omitempty"`
 	Name                string            `yaml:"name,omitempty"`
 	SendExemplars       bool              `yaml:"send_exemplars,omitempty"`
+	SendHistograms      bool              `yaml:"send_histograms,omitempty"`
```

Comment:
I agree, but how about using the name "native histograms"?

I coined the term "sparse histogram" when I came up with them, and the one most important differentiating feature to before was the sparseness. However, not knowing where we came from, it's a weird name, since the "sparse histograms" have so many properties beyond sparseness.

Reply:
There's also the fact that throughout their development, in design docs, PRs, and most notably the two most recent talks, which are probably the most public facing materials, these new histograms have been called "sparse histograms." But of course I respect your judgement.

Reply:
The old histograms weren't really histograms at all. We just put every bucket and the sum and the count of observations in separate, (technically) unrelated time series. And then put a "glue function" (`histogram_quantile`) on top to calculate quantiles (provided the user fed the right unrelated series to the function). You could call that "emulated histograms".

With the new histograms, samples of a single time series can now be a whole histogram rather than just a float. 1st class citizens. Or "native histograms".

WRT prior naming: That's exactly my worry. If we don't change the naming now, we'll be stuck with it, and there will be generations of Prometheus users wondering about the name of these histograms.

Reply:
But I don't think the naming issue is relevant to this PR, so for now maybe we just rename the variable to "SendSparseHistograms" as that's the name that's been used so far, and we can go back and change it later if necessary.

Reply:
I'm already changing names to "native histograms" wherever I can. So my vote is for using "native histogram" for any new thing we add.

Note that `types.proto` already prefers "native histogram".

Reply:
Oh ok, I'll go change it to that then.